deploy/docker/cp-tools/base/rstudio/post_commit.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# Copyright 2017-2019 EPAM Systems, Inc. (https://www.epam.com/)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [[ -f /etc/cp_env.sh ]]; then
+    source /etc/cp_env.sh
+    rm -rf $OWNER_HOME/.rstudio/sessions/active/session-*
```

Comment:
Looks better. But what if OWNER_HOME is not set in the /etc/cp_env.sh (for some weird reason)? It will try to delete  `/.rstudio/sessions/active/session-*` folder which does not exist. Could you please add additional check and fallback to `rm -rf /home/*/.rstudio/sessions/active/session-*` if no OWNER_HOME is available?